src/Mgate/PersonneBundle/Resources/translations/person.fr.yml

```diff
@@ -0,0 +1,155 @@
+generic:
```

Comment:
English is easier for devs when they speak english. When they don't it is an additional hassle.
It's like as if Jeyser was crumbling under contributors.

Code in english if you want, but translations key will be in french. 

Besides it will remove inconsistence in keys (`mail_valid`, `mail_invalid`, `subscribed_newsletter`, `study` instead of project, `mandate` instead of term)